src/main/java/eiteam/esteemedinnovation/commons/handler/GenericTickHandler.java

```diff
@@ -51,7 +50,7 @@
 
     static {
         if (FMLCommonHandler.instance().getSide() == Side.CLIENT) {
-            FMLLog.info("[EI] Getting some fields from reflection for Tick Handling.");
+            EsteemedInnovation.logger.info("[EI] Getting some fields from reflection for Tick Handling.");
```

Comment:
`[EI]` should be removed from this, because it says `[esteemedinnovation]` in each log entry